### ISSUE TITLE
More on alt path streaks

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1068]
a simple solution would be for alt path door to automatically open if you went for mother 2 or 3 run in a row. this way the game would detect you're streaking it

however, it would only auto-open if you have 0 keys. maybe it should even have some way to detect whether or no you 100% explored, although i don't really know if secret rooms and curse rooms should account for that (especially curse rooms since being forced to lose your holy card on T lost because of something like that is just shit)

it's just that this situation with 0 keys on basement 2 happened to me two times in a day already, after i 100% explored and did everything and still haven't gotten a key

i just wish this was looked into somehow because forcing yourself to start over a streak because of it never feels good, and sure I could've just given myself a key with console but that almost always leaves a bad aftertaste — be it for you or other people who might think you're not playing legit or smth 'cause of that, and treat your streak with a lesser dignity

**Comment by Lonrite** [comment]
alternatively, instead of auto opening there could be something like mom's angry / disappointed shadow (ala shovel's shadow when unlocking forgotten) in the starting room - and walking over it makes her run away laughing or something

**Comment by Turtier** [comment]
I posted this comment on a previous post you made regarding alt path, I'm gonna post it here again because I think its a very simple yet effective solution and I want it to be seen, but don't want to open its own suggestion page for it.

Maybe, if you beat the boss of basement 1 / 2 and you don't have any keys, the boss item is guaranteed to be latch key (if you don't have it already). And if you beat the boss of downpour 2 / caves, with less than 2 bombs, its guaranteed to be match book (could arguably still screw you over with troll bombs but that would be pretty unlikely all things considered, unless you gave match book an interaction to also not spawn troll bombs if you have less than 2 bombs).

Just as an extra note, it helps out runs that get key screwed in general. You lose the potential for your boss item to be a damage or tears up, but it'll let you get your item room next floor. Or maybe you intentionally don't pick up keys on floor 1 until after the boss to guarantee a soul heart going into floor 2 (but once again, this sacrifices a potential damage up). Like I think it provides some extra potential strategy outside of the intended purpose of securing alt path despite bad rng.

**Comment by Lonrite** [comment]
yeah I just wish this was looked into somehow, in any way at least - its rly frustrating having to start over your streak because of something like that. imagine if something like that happened on a 20th T lost run - now that's tilting
